src/engine/GraphStoreProtocol.cpp

```diff
@@ -4,23 +4,24 @@
 
 #include "engine/GraphStoreProtocol.h"
 
-#include "parser/SparqlParser.h"
 #include "parser/Tokenizer.h"
 #include "util/http/beast.h"
 
 // ____________________________________________________________________________
 void GraphStoreProtocol::throwUnsupportedMediatype(
     const string_view& mediatype) {
-  throw UnsupportedMediatypeError(absl::StrCat(
-      "Mediatype \"", mediatype,
-      "\" is not supported for SPARQL Graph Store HTTP Protocol in QLever. "
-      "Supported: ",
-      toString(ad_utility::MediaType::turtle), ", ",
-      toString(ad_utility::MediaType::ntriples), "."));
+  throw HttpError(
+      boost::beast::http::status::unsupported_media_type,
+      absl::StrCat(
+          "Mediatype \"", mediatype,
+          "\" is not supported for SPARQL Graph Store HTTP Protocol in QLever. "
+          "Supported: ",
+          toString(ad_utility::MediaType::turtle), ", ",
+          toString(ad_utility::MediaType::ntriples), "."));
```

Comment:
split this up into a separate PR that just consistetnly introduces the `HttpError`.